pages/register/register.service.js

```diff
@@ -33,5 +35,19 @@
 				cb(success, data);
 			});
 		}
+
+		function validateUsernameAndEmail(nickname, email, callback) {
+			api.get(api.url + '/user/exist?name=' + nickname + '&email=' + email, function (data, status) {
+				var success = true;
+
+				if (status >= 400) {
+					console.error('Error while registering');
+					success = false;
+				}
+
+				console.log(data, status);
```

Comment:
Avoid letting logs for things who are not warnings, or important information. This makes the console unreadable and full of thrash 🌵